dotnet/ios/InternSampleLogin/CustomViewController.cs

```diff
@@ -0,0 +1,55 @@
+namespace CodeOnlyDemo;
+
+public class CustomViewController : UIViewController
+{
+    UITextField? usernameField, passwordField;
+    CircleController? circleController;
+
+    public override void ViewDidLoad ()
+    {
+        base.ViewDidLoad();
+
+        if (View is null) 
+            throw new NullReferenceException ();
+
+        View.BackgroundColor = UIColor.Gray;
+
+        nfloat h = 31.0f;
+        nfloat w = View.Bounds.Width;
+
+        usernameField = new UITextField
+        {
+            Placeholder = "Enter your username",
+            BorderStyle = UITextBorderStyle.RoundedRect,
+            Frame = new CGRect(10, 32, w - 20, h),
```

Comment:
```suggestion
            Frame = new CGRect (10, 32, w - 20, h),
```
spaces before the parentheses in this file